File: latihanB/java/Disk.java
public class Disk {
    private String type;
    private int capacity;
    private int price;

    Disk() {}

    Disk(String type, int capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getType() {
        return this.type;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getCapacity() {
        return this.capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    public void printDisk() {
        System.out.println("Type\t\t: " + this.type);
        System.out.println("Capacity\t: " + this.capacity + (this.capacity < 100 ? " TB" : " GB"));
        System.out.println("Price\t\t: " + this.price + " IDR");
    }
}

File: latihanB/python/Disk.py
class Disk:
    def __init__(self, type="", capacity=0, price=0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    def setType(self, type=""):
        self.__type = type

    def getType(self):
        return self.__type

    def setCapacity(self, capacity=0):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    def setPrice(self, price=0):
        self.__price = price

    def getPrice(self):
        return self.__price

    def printDisk(self):
        print("Type\t\t: " + self.__type)
        print("Capacity\t: " + str(self.__capacity) + (" TB" if self.__capacity < 100 else " GB"))
        print("Price\t\t: " + str(self.__price) + " IDR")